docs/machine-learning/ml-tpm-guidance.md

```diff
@@ -0,0 +1,60 @@
+# The role of a TPM in AI projects?
+
+In this document, we explore some of the Program Management considerations for Machine Learning (ML) projects and suggest recommendations for Technical Program Managers (TPM) to effectively work with Data engineering teams.
+
+## Is it an AI problem?
+
+In Artificial Intelligence (AI) projects, the ML component is generally a part of an overall business problem and **NOT** the problem itself. Few considerations for identifying the right fit for your project:
+
+- Determine the value for Machine Learning within the larger scope of a project. Think about the overall business problem first and then evaluate if ML can help address a part of the problem space.
+
+- Engage experts in human experience and employ techniques such as [Design Thinking](https://www.ideou.com/blogs/inspiration/what-is-design-thinking) to **understand the customer needs** and human behavior first. For example, if your customer wants to talk about [Topic Modeling](https://en.wikipedia.org/wiki/Topic_model) instead of [Personas](https://en.wikipedia.org/wiki/Persona_(user_experience)), you are having the wrong conversation.
+
+- Focus on **System Design** principles to identify the architectural components, entities, interfaces, constraints. Ask the right questions early and explore design alternatives with the engineering team.
```

Comment:
We have documented some questions in the envisioning session here: https://github.com/microsoft/code-with-engineering-playbook/blob/ml-tpm-guidance/docs/machine-learning/ml-problem-formulation-envisioning.md - you could all a link to that here if you think it would fit.

Reply:
Thanks, added